user/enterprise.md

```diff
@@ -19,7 +18,7 @@ to use LDAP or SAML to manage your users access. Travis CI Enterprise also
 integrates with GitHub.com, allowing you greater control over your security, and
 to scale out your build infrastructure as needed.
 
-### Why choose Enterprise over travis-ci.com?
+## Why choose Enterprise over travis-ci.com?
 
 Travis CI Enterprise is built to bring the features of the [hosted
```

Comment:
I'd add a clarification in this section or in the top, mentioning that Travis CI Enterprise is their in-house Travis CI solution? I'd hope that would help clarify any confusion until we get visually distinctive docs for hosted and enterprise :) cc/ @danishkhan is there a specific Enterprise wording we should use to do that? :)